en/docs/install-and-setup/setup/api-controller/managing-integrations-with-ctl.md

```diff
@@ -0,0 +1,1397 @@
+            ```
+            ```bash
+            apictl mi get users -p *tester* -e dev
+            ```
+
+    -   **Response**
+
+        ```go
+        USER ID
+        admin
+        capp-tester
+        ```
+
+2.  Get information on a specific user.
+
+    -   **Command**
+        ``` bash
+        apictl mi get users [user-name] -e <environment>
+        ```
+
+        !!! info
+            **Flags:**
+
+            -   Required :  
+                `--environment` or `-e` : Environment of the Micro Integrator to be searched
+            -   Optional :  
+                `--format` : pretty-print using templates
+
+        !!! example
+            ```bash
+            apictl mi get users capp-tester -e dev
+            ```
+
+    -   **Response**
+
+        ```go
+        Name - capp-tester
+        Is Admin - true
+        Roles - Internal/everyone, admin
+        ```
+
+### Add a new user
+
+You can use the command below to add a new user to a Micro Integrator.
+
+-   **Command**
+    ``` bash
+    apictl mi add user [user-name] -e <environment>
+    ```
+
+    !!! info
+        **Flags:**
+
+        -   Required :  
+            `--environment` or `-e` : Environment of the Micro Integrator to be searched
+
+    !!! example
+        ```bash
+        apictl mi add user capp-tester -e dev
+        ```
+
+-   **Response**
+
+    ```go
+    Adding new user [ capp-tester ] status: Added
+    ```
+
+### Delete a user
+
+You can use the command below to remove a user from the Micro Integrator.
+
+-   **Command**
+    ``` bash
+    apictl mi delete user [user-name] -e <environment>
+    ```
+
+    !!! info
+        **Flags:**
+
+        -   Required :  
+            `--environment` or `-e` : Environment of the Micro Integrator to be searched
+
+    !!! example
+        ```bash
+        apictl mi delete user capp-tester -e dev
+        ```
+
+-   **Response**
+
+    ```go
+    Deleting user [ capp-tester ] status: Deleted
+    ```
+
+## Monitor Integration Artifacts
+
+Follow the instructions below to display a list of artifacts or get information about a specific artifact in an environment using CTL:
+
+### Composite Applications (CApps)
+
+1.  List composite applications (CApps) in an environment.
+
+    -   **Command**
+        ``` bash
+        apictl mi get composite-apps -e <environment>
+        ```
+
+        !!! info
+            **Flags:**
+
+            -   Required :  
+                `--environment` or `-e` : Environment of the Micro Integrator to be searched
+            -   Optional :  
+                `--format` : pretty-print using templates
+
+        !!! example
+            ```bash
+            apictl mi get composite-apps -e dev
+            ```
+
+    -   **Response**
+
+        ```go
+        NAME                            VERSION
+        HealthCareCompositeExporter     1.0.0
+        FoodServiceCApp                 2.0.0
+        ```
+
+2.  Get information on a specific composite application in an environment.
```

Comment:
@SanojPunchihew, why is this numbered as 2. Numbering is used to denote steps that need to be carried out in sequential order. I do not think that step 1 and 2 have a dependency and needs to be executed in this order please clarify.

Reply:
The user doesn't need to execute in this order. I added like that thinking the user may want to list the artifacts first and select one of them to retrieve more information about it. Maybe we can change the numbering to a more convenient listing type since it's not mandatory run them in sequential order.